### DISCUSSION TITLE
Automatically stop playing blackgraund music

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1606] [Q&A] [answered]
I am not able to understand well the dynamics of this option of the time start / after. One side talks in seconds and the other minutes. I consulted the M3 document, but the doubt remains. For example, my meeting starts at 7 pm, I want the music to start as soon as I turn on the m3 and finish 1 minute earlier. **How should this time setting look?**

![image](https://user-images.githubusercontent.com/90984283/222576381-6bd2d871-01c2-4b09-b064-c094c2b002dc.png)

**Comment by mtdvlpr** [comment, marked as answer]
Hi @e-Feitosa, when you select seconds, the music will stop xx seconds before the meeting. When you select minutes, the music will stop after xx minutes. Suppose you start the music at 6:30 pm. When you select (60) seconds, the music stops at 6:59 pm (60 seconds before the meeting). When you select (60) minutes, the music stops at 7:30 pm (60 minutes after you've started the music).

**Reply by e-Feitosa** [reply]
Hi, @mtdvlpr, dreat! Thanks for the explanations. What generated a doubt here was the following, if the meeting is today at 7 pm, if I open the M3 at 3 pm, the music does not start automatically. The music is only starting automatically 1 hour before. Here I need it like this for example: Meeting 19:30, music should stop 19:29. This way works well as long as the PC is turned on after 6:30 pm. What was confusing was that we understood that regardless of the time the M3 opened on the day of the meeting, the music would start and stop 60 seconds before.

![image](https://user-images.githubusercontent.com/90984283/222746882-7e46ddeb-cc49-4671-987b-b24174c2076d.png)
